Resources/AzureDevOps/StepTemplates/deploy-azure-function.yml

```diff
@@ -0,0 +1,50 @@
+parameters:
+  FunctionType: ''
+  AzureSubscription: 'SFA-CDH-Dev/Test (962cae10-2950-412a-93e3-d8ae92b17896)'
+  SharedResourceGroup: ''
+  ApimServiceName: ''
+  ApimImportScript: ''
+  SpecificationPath: ''
+  WebAppName: ''
+  ResourceGroup: ''
+  FunctionAppDomain: ''
+  PackageFile: ''
+
+steps:
+- task: AzureRmWebAppDeployment@4
+  displayName: 'Azure App Service Deploy: ${{ parameters.WebAppName }}'
+  inputs:
+    azureSubscription: '${{ parameters.AzureSubscription }}'
+    appType: functionApp
+    WebAppName: '${{ parameters.WebAppName }}'
+    Package: "${{ parameters.PackageFile }}"
+    DeployToSlotFlag: true
+    ResourceGroupName: '${{ parameters.ResourceGroup }}'
+    SlotName: staging
+
+- task: AzureAppServiceManage@0
+  displayName: 'Swap Slots: ${{ parameters.WebAppName }}'
+  inputs:
+    azureSubscription: '${{ parameters.AzureSubscription }}'
+    WebAppName: '${{ parameters.WebAppName }}'
+    ResourceGroupName: '${{ parameters.ResourceGroup }}'
+    SourceSlot: staging
+
+- powershell: |
```

Comment:
For consistency, can we decide if we are using a pipe at end of the powershell inline shortcut for a single line or not